WasatchDemo.m

```diff
@@ -19,6 +19,16 @@
 % open the first spectrometer found
 spectrometer = driver.getSpectrometer(0);
 
+% set isAndor 'true' to load a virtual EEPROM file for an XL-Series 
+% spectrometer with Andor camera
+isAndor = false;
```

Comment:
This seems dependent on https://github.com/WasatchPhotonics/Wasatch.NET/pull/113#issuecomment-1735979076

Reply:
The other PR you mention would allow us to further simplify this MATLAB demo.  Basically, it will let me replace this:
```
% user has to manually change this variable
isAndor = false
if isAndor
    ...
```
...with this:
```
% WasatchNET already knows if the spectrometer has an Andor detector
if spectrometer.isAndor
    ...
```

Reply:
Ah so this is code that the user would get to skim over before running?

I saw `isAndor = false;` and figured that wouldn't be seen by any end user. As you can see, I'm not much of a Matlab user.